docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* Add with_rendered_component_in_browser method to enable testing JS interactions in components
```

Comment:
```suggestion
* Add `with_rendered_component_path` method to enable testing JS interactions in components
```

I thought I made this suggestion already, but maybe I didn't 😅 I think we should make it clear that we are not providing a browser.